pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
```

Comment:
Also, I think the first example should be the simplest: print to the default printer, using simply `lp` with no parameters other than the filename (with the note that the default printer can be found by running `lpstat -d`).